scvi/module/base/_base_module.py

```diff
@@ -197,6 +201,11 @@ def inference(
         """
         pass
 
+    @abstractmethod
+    def inference_no_encode(self, qz_m, qz_v, latent_data_type, n_samples=1):
+        """Runs the inference routine but skips encoding, used in latent mode."""
+        pass
+
```

Comment:
this is going to break a lot of peoples code, we should not make this abstract and instead can just add a notimplementederror?